evm/README.md

```diff
@@ -18,7 +18,16 @@ You also need to create a `info/keys.json` file. See `evm/.example.keys.json` fo
 4. Add additional params in `.env` such as admin addresses, governance etc.
 5. If you'd like to auto-verify the contract on the explorer, then add the explorer API key under `info/keys.json` (see `.example.keys.json`), and add `--verify` flag
 6. Run the following depending on the service,
-`node evm/deploy-gateway-v5.x.js --env testnet -n fantom`
+`node evm/deploy-gateway-v5.0.x.js --env testnet -n fantom`
+
+#### Gateway Upgrade
```

Comment:
This covers v6.1 -> v6.1 upgrade. v4.3 -> v6.1 upgrade is similar (need to deploy v4.3 gateway first).

A subsequent update will cover the upgrade tx when using InterchainGovernance